app/src/main/java/io/github/droidkaigi/confsched2018/util/ext/DateExt.kt

```diff
@@ -6,13 +6,16 @@ import java.util.Date
 import java.util.Locale
 import java.util.TimeZone
 
+private val dateFormat = SimpleDateFormat("MM/dd", Locale.US)
```

Comment:
What do you think about this pr?
https://github.com/DroidKaigi/conference-app-2018/pull/97

Reply:
oh... well, the current implementation is thread-safe but high-cost.
Let me refactor again to be thread-safe.